build/docker/script/run

```diff
@@ -26,7 +26,14 @@ subr_start_over() {
 }
 
 subr_build() {
-    ./build/dependency mk
+    ./autogen.sh
+    ./configure --disable-shared $CONFIGURE_FLAGS
+    make V=0 -j$(nproc) $MAKE_CHECK_TARGET || {
+        if [ -f ./test-suite.log ]; then
+            cat ./test-suite.log
+        fi
+        exit 1
+    }
```

Comment:
It was backwards to use the script for building `mk` on Android/iOS to drive CI. Since changes in the former may lead to changes in the latter, like in this case. Rather, just roll out the command line for testing MK.

Reply:
Motivation for the change: rolling out explicitly what to do is much more robust.